### DISCUSSION TITLE
Display previews from `https://preview.redd.it/` domain in feed

### DISCUSSION THREAD

**Discussion by soredake** [Docile-Alligator/Infinity-For-Reddit#1639] [Ideas]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/ideas) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/completed-ideas) ideas to see if someone else has already submitted the same request.
- [X] I will describe the feature with as much detail as possible.

### Description

Right now previews from this domain is not shown in feed:
![image](https://github.com/Docile-Alligator/Infinity-For-Reddit/assets/5204968/de5eaf20-2ae5-4428-a4ee-ec8bf27cc1d7)

### Additional context

Post for testing https://www.reddit.com/r/Kaguya_sama/comments/1c6no2i/kaguya_shinomiya_and_miyuki_shirogane_watching/

### How important is this feature?

Nice to have

**Comment by Docile-Alligator** [comment]
Please update your app. This is already available in v6.7.0.

**Reply by soredake** [reply]
@Docile-Alligator screenshot was taken on 7.1.1 (latest) version

**Comment by soredake** [comment]
Created bug report https://github.com/Docile-Alligator/Infinity-For-Reddit/issues/1641
